websites/W/Watch-Anime/presence.ts

```diff
@@ -0,0 +1,93 @@
+//#region FUNCTIONS
+interface AnimeInfo {
+	img: string;
+	name: string;
+}
+
+async function getInformationAnime(
+	nameAnime: string
+): Promise<AnimeInfo | null> {
+	try {
+		const response = await fetch(
+				`https://api.watch-anime.fr/getAnime/${nameAnime}`
+			),
+			data = await response.json();
+
+		if (data.length > 0) {
+			return {
+				img: data[0].affiche_anime,
+				name: data[0].nom_anime,
+			};
+		}
```

Comment:
Considering how this function is used, consider creating a cache to avoid sending unnecessary amounts of HTTP requests to the website.